root/static/scripts/account/components/EditProfileForm.js

```diff
@@ -67,7 +67,7 @@ const genderOptions = {
   options: [
     {label: N_l('Male'), value: 1},
     {label: N_l('Female'), value: 2},
-    {label: N_l('Other'), value: 3},
+    {label: N_lp('Other', 'gender option'), value: 3},
```

Comment:
I'd expect this should be `() => lp_attributes('Other', 'gender')` instead (and similarly for the two above).

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/1004/commits/eda986d60a635c0fc7d1b9deb06e03b1f6dcee21, re-updated pot files in https://github.com/metabrainz/musicbrainz-server/pull/1004/commits/a6a90c291a71b0ab275918f034638b88a0400c23 (no more gender in server resource).